core/input/input.cpp

```diff
@@ -121,6 +121,10 @@ void Input::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_accelerometer"), &Input::get_accelerometer);
 	ClassDB::bind_method(D_METHOD("get_magnetometer"), &Input::get_magnetometer);
 	ClassDB::bind_method(D_METHOD("get_gyroscope"), &Input::get_gyroscope);
+	ClassDB::bind_method(D_METHOD("start_gps_tracker", "time_ms", "distance"), &Input::start_gps_tracker);
+	ClassDB::bind_method(D_METHOD("stop_gps_tracker"), &Input::stop_gps_tracker);
+	ClassDB::bind_method(D_METHOD("get_location"), &Input::get_location);
+	ClassDB::bind_method(D_METHOD("get_altitude"), &Input::get_altitude);
```

Comment:
Remember to run `--doctool` with a compiled editor binary to [update the class reference XML](https://docs.godotengine.org/en/latest/community/contributing/updating_the_class_reference.html), then add a description in `doc/classes/Input.xml` for the newly exposed methods.

Reply:
Added, thanks.